api/hasura/cron/epochs.ts

```diff
@@ -354,7 +354,6 @@ export async function notifyEpochEnd({
                 0
               ),
               userCount: circle.users.length,
-              circleHistoryLink: `https://app.coordinape.com/circles/${circle.id}/history`,
```

Comment:
We can build this link in the discord bot instead now that we're sending the circle id